destination_test.go

```diff
@@ -2,24 +2,11 @@ package connectorname_test
 
 import (
 	"context"
-	"strings"
 	"testing"
 
 	connectorname "github.com/conduitio/conduit-connector-connectorname"
 )
 
-func TestConfigureDestination_FailsWhenConfigEmpty(t *testing.T) {
```

Comment:
This test doesn't make sense anymore since the SDK will validate the config before it reaches `Configure`.